core/arch/arm/plat-rockchip/psci_rk322x.c

```diff
@@ -43,6 +43,175 @@
 #include <tee/entry_std.h>
 #include <tee/entry_fast.h>
 
+struct dram_data {
+	uint32_t cru_mode_con;
+	uint32_t cru_clksel0;
+	uint32_t cru_clksel1;
+	uint32_t cru_clksel10;
+	uint32_t cru_clksel21;
+	uint32_t cru_clkgate[CRU_CLKGATE_CON_CNT];
+};
+
+static struct dram_data dram_d;
+
+static uint32_t clks_gating_table[CRU_CLKGATE_CON_CNT] = {
+	/* gate: 0-3 */
+	0xefb8,
+	0x0ff7,
+	0xfff4,
+	0x887f,
+	/* gate: 4-7 */
+	0x0030,
+	0x00f8,
+	0x07e0,
+	0xc000,
+	/* gate: 8-11 */
+	0xff84,
+	0xb047,
+	0x1ca0,
+	0x57ff,
+	/* gate: 12-15 */
+	0x0000,
+	0x00ff,
+	0x1cc0,
+	0x000f,
+};
+
+static void clks_gating(void)
```

Comment:
It's usually preferred to use a verb for functions that perform some action, what about `clks_disable(void)`?